docs/changehistory/NextVersion.md

```diff
@@ -261,6 +262,12 @@ UI item provider types no longer extend from `ProviderItem`.
 - Fixed to correctly update options of a `Select` component after store changes.
 - Fallback selection scope labels are removed and no longer applied.
 
+### Components relocated
+
+[ModelsTree]($appui-react) and [CategoryTree]($appui-react) were moved to [@itwin/tree-widget-react](https://github.com/iTwin/viewer-components-react/tree/master/packages/itwin/tree-widget) package and deprecated in the `@itwin/appui-react` package 3.6 release. They were removed from `@itwin/appui-react` in 4.0.
+
+[SpatialContainmentTree]($appui-react) were deprecated in favor of `SpatialContainmentTree` from [@itwin/breakdown-trees-react](https://github.com/iTwin/viewer-components-react/tree/master/packages/itwin/breakdown-trees) package in the 3.6 release. [SpatialContainmentTree]($appui-react) was removed from `@itwin/appui-react` in 4.0.
```

Comment:
fyi @calebmshafer @JValiunas @grigasp 

We need to figure out whats going on with the spatial trees moving fwd before we release

Reply:
Before we release what ?

Reply:
if the other trees were moved into tree-widget then i guess if we need to resurrect the spatial tree, it can live in there. 
created separate issue to track it.